tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -42,7 +42,7 @@ spec:
       description: Success if the task succeeds, the error otherwise
   steps:
     - name: pull-and-push-images
-      image: quay.io/konflux-ci/release-service-utils:6556e8a6b031c1aad4f0472703fd121a6e1cd45d
+      image: quay.io/konflux-ci/release-service-utils:7835e32b1974f956a6c942e24adbc79705cab12e
```

Comment:
Is this just https://github.com/konflux-ci/release-service-catalog/pull/768? I assume it is here to make your testing work? I don't think it should be part of the PR to be merged though (would make things easier if the e2e was passing so 768 would merge and your rebase would pull it in)

Reply:
no. this was for https://github.com/konflux-ci/release-service-utils/commit/7835e32b1974f956a6c942e24adbc79705cab12e